components/Search.tsx

```diff
@@ -0,0 +1,59 @@
+"use client";
+
+import { FormEvent, useState } from "react";
+import { Button } from "@/components/ui/button";
+import { Input } from "@/components/ui/input";
+import FormationCard from "@/components/FormationCard";
+import { ScrollArea } from "@/components/ui/scroll-area";
+import { Separator } from "@/components/ui/separator";
+import { Label } from "@/components/ui/label";
```

Comment:
Maybe a future refactor, but maybe at the base of `"@/components/ui"` we can export all components from a single `index.ts` file then import like so

```
import { Button, Input, ScrollArea, ... } from "@/components/ui"
```